### ISSUE TITLE
Add support for leisure=schoolyard

### ISSUE THREAD

**Issue by coolultra1** [gravitystorm/openstreetmap-carto#4491]
With [1,7K uses](https://taginfo.openstreetmap.org/tags/?key=leisure&value=schoolyard), [**leisure=schoolyard**](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dschoolyard) is a tag in good use.

I think, either rendering the area colored, similiar to **highway=pedestrian** or with an icon would be appropiate.

Thank you.

**Comment by imagico** [comment]
About half of the uses are in western Europe, use elsewhere is very sporadic.

Quality and consistency of mapping is mostly decent but there sometimes is a tendency to use the tag for collective mapping of all outdoor areas around a school independent of their actual use - like:

https://www.openstreetmap.org/way/168031544
https://www.openstreetmap.org/way/920577948
https://www.openstreetmap.org/way/162774233
https://www.openstreetmap.org/way/33150065

**Comment by Lukas458** [comment]
> Quality and consistency of mapping is mostly decent but there sometimes is a tendency to use the tag for collective mapping of all outdoor areas around a school independent of their actual use

Yes, for sure. This would rather describe a landuse for me then, so landuse=school for example. But leisure=schoolyard should be used only for the area that's the schoolyard where kids play actually.
But sometimes it is very difficult to distinguish between the different areas and their usage on the school's "campus".
